extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientRecorder.java

```diff
@@ -99,6 +100,10 @@ public ConnectionPoolListener get() {
         };
     }
 
+    public static String getMicrometerCommandListenerClassName() {
+        return MicrometerCommandListener.class.getName();
+    }
+
```

Comment:
As this is used in a single place you can directly add the class name at the use site.
This would also avoid loading the class at runtime if not used wich would be better.

Reply:
> As this is used in a single place you can directly add the class name at the use site. This would also avoid loading the class at runtime if not used wich would be better.

micrometer wasn't on class path in deployment module, that's why I've put it there. I'll add optional dependency and move the code to the deployment